docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -1,7 +1,16 @@
-# GNA device {#openvino_docs_OV_UG_supported_plugins_GNA}
-## Introducing the GNA Plugin
+# GNA device {#openvino_docs_IE_DG_supported_plugins_GNA}
```

Comment:
looks like we need to keep `openvino_docs_OV_UG_supported_plugins_GNA`

Reply:
change included. With this shift we should probably also change the VPU.